pkg/tests/files.go

```diff
@@ -53,3 +54,28 @@ func AssertFilesExist(t *testing.T, expected bool, paths ...string) {
 		}
 	}
 }
+
+func AssertEqualFileText(t *testing.T, expectedFile string, actualFile string) error {
```

Comment:
I think, we can replace this file test helpers with `https://godoc.org/gotest.tools/fs`.

Reply:
awesome, I didn't know about those - will do thanks!

Reply:
lemme refactor the unit tests to reuse gotest.tools in a separate PR...